reference/content/driver-reactive/tutorials/commands.md

```diff
@@ -42,17 +42,15 @@ MongoDatabase database = mongoClient.getDatabase("test");
 
 For additional information on connecting to MongoDB, see [Connect to MongoDB]({{< ref "connect-to-mongodb.md" >}}).
 
-## Run `buildInfo` and `collStats` Commands
+## Run the `buildInfo` Command
 
-To run a command, construct a [`Document`]({{< apiref "bson" "org/bson/Document.html" >}})
+To run the `buildInfo` command, construct a [`Document`]({{< apiref "bson" "org/bson/Document.html" >}})
 object that specifies the command and pass it to the `runCommand()` method.
 
-The following runs the [`buildInfo`]({{<docsref "reference/command/buildInfo" >}}) command and the [`collStats`]({{<docsref "reference/command/collStats" >}}) command:
+The following runs the [`buildInfo`]({{<docsref "reference/command/buildInfo" >}}) command and prints the results:
```

Comment:
I: [Per Style Guide, avoid "following" as a noun](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#f:~:text=the%20follow%20up.-,following,-preceding%20Use%20these)

Applies to all examples.

```suggestion
The following code example runs the [`buildInfo`]({{<docsref "reference/command/buildInfo" >}}) command and prints the results:
```